demoproductform/src/CQRS/CommandBuilder/ModuleProductCommandsBuilder.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * 2007-2020 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0).
+ * It is also available through the world-wide-web at this URL: https://opensource.org/licenses/AFL-3.0
+ */
+declare(strict_types=1);
+
+namespace PrestaShop\Module\DemoProductForm\CQRS\CommandBuilder;
+
+use PrestaShop\Module\DemoProductForm\CQRS\Command\SaveMyModuleCustomFieldCommand;
+use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
+use PrestaShop\PrestaShop\Core\Form\IdentifiableObject\CommandBuilder\Product\ProductCommandsBuilderInterface;
+
+/**
+ * This class is responsible for building cqrs commands from product form data.
+ * Once you tag this service as "core.product_command_builder" (check module services.yml)
+ * the core ProductCommandsBuilder will start using this builder
+ *
+ * Don't forget you can also handle your custom fields like any other identifiable object
```

Comment:
I think it will be very important to write this in the documentation that explains this module content, scope and context.

_This is one way to do it but you it is not the only way_

We don't want people to start complaining "it's so hard to extend the product form, now ! you have to create a Command, a CommandHandler... it was so much easy before!"